src/main/java/com/lgtm/easymoney/configs/Consts.java

```diff
@@ -0,0 +1,21 @@
+package com.lgtm.easymoney.configs;
+
+import java.util.Map;
+
+public class Consts {
+    public static final String USER_EMAIL_CONSTRAINT = "user_email_uniq_idx";
```

Comment:
nice:) it's good to put constants in one place. 
seems like this consts is currently only used for checking db constraints, consider renaming to make it more specific? if this later would be reused to store other constants, add more comments to document its usage

cc: @jenniferduan45  @howieraem  @KenXiong123

Reply:
You mean rename the class or?